docs/index.html

```diff
@@ -39,11 +39,11 @@
       search: {
         paths: 'auto',
         placeholder: {
-          '/zh-cn/': '搜索',
+          '/zh-cn/': '搜索', '/hi/': 'खोजें',
           '/': 'Search'
         },
         noData: {
-          '/zh-cn/': '没有结果!',
+          '/zh-cn/': '没有结果!', '/hi/': 'कोई परिणाम नहीं',
```

Comment:
please change this country code to the correct one for india and hindi

Reply:
actually on 2nd thoughts. Add another entry, don't replace the Chinese one please.

Reply:
ok, please format this correctly. I mistakenly thought it wasn't set up correctly. Add a CR